public/app/core/logs_model.ts

```diff
@@ -149,13 +149,18 @@ export function makeSeriesForLogs(rows: LogRowModel[], intervalMs: number, timeZ
     });
 
     const timeField = data.fields[1];
-
     timeField.display = getDisplayProcessor({
       config: timeField.config,
       type: timeField.type,
       isUtc: timeZone === 'utc',
     });
 
+    const valueField = data.fields[0];
+    valueField.config = {
+      ...valueField.config,
```

Comment:
Could `valueField.config` ever contain `color`? Then it would be overridden here.

Reply:
Only if we change the logic of how data frames are created from query result.  Now the flow is query result -> `dataFrameToLogsModel` -> `makeSeriesForLogs`.  If something happens in the reducer before `dataFrameToLogsModel` is called then